list/pop_back.md

```diff
@@ -0,0 +1,43 @@
+# pop_back
+
+**Description :** The pop_back() function removes an element from the back of a list container. When used, it decreases the size of the container by 1.
+
+**Example** :
+```cpp
+#include <iostream>
+#include <list>
+
+int main(){
+    //declare an empty list
+    std::list<int>mylist; 
+    
+    //add elements at the back using push_back
+    mylist.push_back(2);
+    mylist.push_back(3);
+    mylist.push_back(5);
+    mylist.push_back(8);
+  
+    //print elements of list
+    std::cout << '\n'; //cosmetic
+    std::cout << "  List elements; BEFORE pop from back \n";
+    
+    //pre-increment counter per style guide
+    for (auto it = mylist.begin(); it != mylist.end(); ++it){ 
+        std::cout << "  " << *it << '\n'; 
+    }
+
+    //pop element from back 
+    mylist.pop_back();
+
+    //print elements of  list
+    std::cout << '\n'; //cosmetic
+    std::cout << "  List elements; AFTER pop from back \n";
+    for (auto it = mylist.begin(); it != mylist.end(); ++it){ 
+        std::cout << "  " << *it << '\n'; 
+    }
+    std::cout << '\n'; //cosmetic
+
+    return 0;
+}
+```
+**[Run Code](https://rextester.com/FFW41893)**
```

Comment:
Newline here

Reply:
Ah - now I see, I need to make the changes. Happy to do so here. I just need more clarification on the last two comments.

Reply:
For clarification, are you saying to remove the newline on line 38 (because I thought it just *looked* better with the newline) OR are you saying I need a carriage return at some point in line 42 perhaps?

Reply:
I implemented the suggested changes. It doesn't look like I needed to do another pull request. Let me know if updates are good and I submitted correctly. Thank you!